ncm-metaconfig/src/main/metaconfig/libvirtd/libvirtd_level1.tt

```diff
@@ -0,0 +1,19 @@
+[%- libvirtd_list = ['access_drivers', 'tls_allowed_dn_list', 'sasl_allowed_username_list'] -%]
```

Comment:
better to test the type infromatio like https://github.com/quattor/configuration-modules-core/pull/620/files#diff-2f4e86030605ac8ab2ee8ce3061b71e5R1